### PR TITLE
Add super-collections

### PR DESCRIPTION
**Pull request by hoechenberger**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hoechenberger** [comment]
@conda-forge-admin, please rerender

**Comment by hoechenberger** [comment]
@conda-forge/help-python, ready for review :)

File: recipes/super-collections/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: 94c1ec96c0a0d5e8e7d389ed8cde6882ac246940507c5e6b86e91945c2968d46
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (ocefpaf) [inline recipes/super-collections/meta.yaml:15]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/super-collections/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% set name = "super-collections" %}
+{% set pypi_name = "super_collections" %}
```
Comment (ocefpaf) [inline recipes/super-collections/meta.yaml:2]: PyPI uses both names, in conda, there is only one. This extra jinja is confusing and what we recommend is:

Use the underscore version, b/c that matches the new URL in PyPI. Or add a multiple output to create both recipes. Here is an example: https://github.com/conda-forge/ioos_metrics-feedstock/blob/main/recipe/meta.yaml

**Comment by hoechenberger** [comment]
@ocefpaf Thanks for your review! I have tried to address all of your comments.

**Pull request merged**